### DISCUSSION TITLE
Connecting more than two texture to make a Fiber: f=fibre(ori1, ori2,ori3);

### DISCUSSION THREAD

**Discussion by snayir6** [mtex-toolbox/mtex#1154] [Ask Anything]
Hi Ralf and Rudiger,
Would you let me know that if there is a way to connect more than 2 textures to make a fiber? I think it only makes fibers out of or1 and ori3 and ignores the one in the middle

```
% define crystal and specimen symmetry
cs = crystalSymmetry('432');
ss = specimenSymmetry('1');

% and two orientations
ori1 = orientation.cube(cs,ss);
ori2 = orientation.goss(cs,ss);
ori3 = orientation.brass(cs,ss);

%Make fiber from all three orientation
f=fibre(ori1, ori2,ori3);
```
Is it possible to make fiber from all 3 texture components? I appreciate the help in advance
Thanks,
Selda

**Comment by ralfHielscher** [comment]
What is wrong with
```
f1=fibre(ori1, ori2);
f2=fibre(ori2,ori3);
```

**Reply by snayir6** [reply]
Hi Ralf,
I want to make a single fiber out of 3 or more textures, is it possible. Only connecting two textures is not correct when I define the alpha fiber. would you let me know if there is a way to connect more textures to make a single fiber?
Thanks,
Selda

![image](https://user-images.githubusercontent.com/51959635/126789519-1830b500-f617-4732-9da4-fd910aecdae3.png)

An example alpha fiber
![image](https://user-images.githubusercontent.com/51959635/126789725-b2e039d9-63b2-4501-aaab-1ec91882da04.png)

**Reply by ralfHielscher** [reply]
I guess you need to explain more in detail what you want to achieve.
For me a fibre connecting two orientations is an arc in orientation space with the two orientations as starting and end point. If you have three orientations you can have an arc from ori1 to ori2 and an arc from ori2 to ori3. This is what my answer was about.

You meant something different. But its hard to understant: "I want to make a single fiber out of 3 or more textures"

What does mean "make"? In general there is no fibre passing though three arbitrary orientations. Similarly as for three points on the sphere the is not always a big circle passing through them.
Ralf.

**Reply by snayir6** [reply]
Hi Ralf, sorry for my vagueness,
The usual definition of alpha fiber is a combination of Goss+Brass+A+Rotated goss. However, the default alpha fiber in mtex is a combination of rotated cube+ E. Your definition of fiber is probably correct but when I read the papers, alpha fiber is a combination of more than 3 textures. 
I think we can improve the fibers in mtex by adding the ability to connect more than 2 textures and plotting them specifically according to the Phi1 axis etc. But I could not figure out yet

Here is an example of alpha fiber for more clarification. Thanks for your help in advance, if I will figure it out, I will write to you
Thanks,
Selda

![image](https://user-images.githubusercontent.com/51959635/126810327-e883dd72-1381-4935-a4d5-0eca617673cf.png)
REF: Effect of stacking fault energy on deformation texture development of nanostructured materials produced by the ARB process RoohollahJamaati n, MohammadRezaToroghinejad
